pynuodb/crypt.py

```diff
@@ -23,6 +22,8 @@
 import random
 import binascii
 import sys
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
```

Comment:
Is it worthwhile to put this into a try/catch and fall back to the open-coded implementation if we can't load this?

Remember that just because our nuopython contains these, other customers might not have it.  I don't know if it's something we want to enforce that everyone has, even if they don't need to run diagnose and so don't really need the performance...?

Reply:
Yeah that sounds okay with me. If we do that, should cryptography be removed from requirements.txt and setup.py?